suite/src/__tests__/fast/longevity.test.ts

```diff
@@ -12,9 +12,9 @@ const ComposeDbUrls = String(process.env.COMPOSEDB_URLS).split(',')
  * accessible days later, possibly even after a new version of Ceramic is released and deployed.
  */
 describe('longevity', () => {
-  // `step` is incremented once for each r/w node during the `update` test, so use
-  // the length of the r/w URL array to test the content.
-  const expectedContent = { step: ComposeDbUrls.length }
+  // `step` is incremented once for the first r/w node and twice for each subsequent node,
+  // during the `update` test, so use the length of the r/w URL array to test the content.
+  const expectedContent = { step: 1 + (ComposeDbUrls.length - 1) * 2 }
```

Comment:
note this is going to break the longevity tests for existing streams in the database that were created with the old version of the update test that had fewer steps.  We might need to coordinate merging this PR with clearing out the dynamodb database to prevent spurious failures, unfortunately